nixos/modules/programs/git.nix

```diff
@@ -58,6 +58,10 @@ in
         '';
       };
 
+      prompt = {
+        enable = mkEnableOption "automatically sourcing git-prompt.sh";
```

Comment:
Perhaps extend this description a bit?

`automatically sourcing git-prompt.sh. This does not change $PS1 by itself, but makes the functions for putting git info into $PS1 available`

That said, I'm not sure this module is worth it? You have to configure $PS1 somewhere, and why not source git-prompt.sh there?